airbyte-workers/src/main/java/io/airbyte/workers/process/KubePortManagerSingleton.java

```diff
@@ -49,8 +51,8 @@ public static synchronized KubePortManagerSingleton getInstance() {
    * getInstance().
    */
   public static synchronized void init(final Set<Integer> ports) {
-    if (instance != null) {
-      throw new RuntimeException("Cannot initialize twice!");
+    if (instance != null && Sets.intersection(instance.getAllPorts(), ports).size() == ports.size()) {
```

Comment:
should this be 
```suggestion
    if (instance != null && Sets.intersection(instance.getAllPorts(), ports).size() != ports.size()) {
```

the check here should be if the intersection is not complete right?

Reply:
does this mean we are ok with multiple initialisation with different port sets?